tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -16,6 +16,15 @@ open Xunit
 
 type InteractiveTests() =
 
+    [<Fact>]
+    member _.``ValueRestriction error message should not have type variables fully solved``() =
+        use script = new FSharpScript()
+        let code = "id id"
+        let _, errors = script.Eval(code)
+        Assert.Equal(1, errors.Length)
+        let msg = errors[0].Message
+        Assert.DoesNotMatch("obj -> obj", msg)
```

Comment:
Should we rather pin down what the message actually is?

Reply:
Good point, I forgot to revist this. I was just wondering whether it should be a hardcoded `'_a -> '_a` or some regex that allows for different names of type variables